src/server.cpp

```diff
@@ -2938,10 +2940,9 @@ void Server::UpdateCrafting(RemotePlayer *player)
 	if (!clist || clist->getSize() == 0)
 		return;
 
-	if (!clist->checkModified()) {
-		verbosestream << "Skip Server::UpdateCrafting(): list unmodified" << std::endl;
+	if (!clist->checkModified())
 		return;
-	}
+	verbosestream << "Server: Updating crafting" << std::endl;
```

Comment:
There's a visual feedback when this happens: updated craft preview slot. The previous log message is used to determine how many calls that can be skipped for speed optimization (incremental inventory).

Reply:
So you are saying this should be removed?